gaek/ndb_json.py

```diff
@@ -128,8 +127,16 @@ def default(self, obj):
 
     obj_type = type(obj)
     # NDB Models return a repr to calls from type().
-    if obj_type not in NDB_TYPE_ENCODING and hasattr(obj, '__metaclass__'):
-      obj_type = obj.__metaclass__
+    if obj_type not in NDB_TYPE_ENCODING:
+      if hasattr(obj, '__metaclass__'):
+        obj_type = obj.__metaclass__
+      else:
+        # Try to encode subclasses of types
+        for ndb_type in NDB_TYPE_ENCODING.keys():
```

Comment:
Good point-- making it deterministic will already help a lot.  I'll update this.